src/bitmessageqt/tests/support.py

```diff
@@ -12,6 +13,10 @@ class TestSupport(TestBase):
     SUPPORT_ADDRESS = 'BM-2cUdgkDDAahwPAU6oD2A7DnjqZz3hgY832'
     SUPPORT_SUBJECT = 'Support request'
 
+    def tearDown(self):
+        """Delete the support address from addressbook"""
+        delete_from_addressbook(self.SUPPORT_ADDRESS)
```

Comment:
core tests run from appdata

Reply:
Maybe yes, but noone is going to do that.

Reply:
I did it in one of my rebased branches. It did not help, because cleanup is the least part of the problem.

Reply:
Did you try this?
```patch
--- a/src/tests/test_process.py
+++ b/src/tests/test_process.py
@@ -33,6 +33,7 @@ class TestProcessProto(unittest.TestCase):
     def setUpClass(cls):
         """Setup environment and start pybitmessage"""
         cls.home = os.environ['BITMESSAGE_HOME'] = tempfile.gettempdir()
+        cls._cleanup_files()
         put_signal_file(cls.home, 'unittest.lock')
         subprocess.call(cls._process_cmd)  # nosec
         time.sleep(5)

```

Reply:
ok let me try